src/containers/GuestInfoContainer.tsx

```diff
@@ -0,0 +1,176 @@
+      .required('Birthdate is required')
+  });
+
+  const [privacy, setPrivacy] = useState<boolean>(false);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [error, setError] = useState<undefined | string>();
+
+  const methods = useForm<PersonalInfo>({
+    resolver: yupResolver(NewUserSchema),
+    defaultValues
+  });
+
+  const {
+    watch,
+    handleSubmit,
+    formState: { isSubmitting }
+  } = methods;
+
+  const values = watch();
+
+  const onSubmit = useCallback(async () => {
+    logger.info('submit user data', values);
+    try {
+      setError(undefined);
+      setLoading(true);
+
+      if (checkout === undefined) {
+        throw Error('Priced offer undefined');
+      }
+
+      if (!checkout.offerId) {
+        throw Error('Priced offer offerId not defined');
+      }
+
+      //todo improve typings
+      const date = new Date(values.birthdate as unknown as string);
+      const formattedDate = DateTime.fromObject({
+        year: date.getFullYear(),
+        month: date.getMonth() + 1, //js month starts from 0
+        day: date.getDate(),
+        hour: 0,
+        minute: 0
+      }).setZone('utc');
+      values.birthdate = new Date(formattedDate.toISO());
+
+      await axios.request(new PersonalInfoRequest(checkout.offerId, values));
+
+      dispatch({
+        type: 'SET_CHECKOUT',
+        payload: {
+          ...checkout,
+          personalInfo: values
+        }
+      });
+
+      logger.info('Guest info sent successfully');
+      navigate('/checkout/' + checkout.offerId);
+    } catch (error) {
+      const message = (error as Error).message || 'Unknown useAuthRequest error';
+      setLoading(false);
+      setError(message);
+    }
+  }, [values, checkout]);
+
+  return (
+    <FormProvider methods={methods} onSubmit={handleSubmit(onSubmit)}>
+      <Grid justifyContent="center" container spacing={3}>
+        <Grid item xs={12} md={6}>
+          <Typography>
+            This Information will be strictly used by the hotel and win.so to confirm your
+            booking! It is mandatory in order for the hotel to reserve your room.
+          </Typography>
+          <Card sx={{ my: 3, p: 3 }}>
+            <Box
+              sx={{
+                display: 'grid',
+                columnGap: 2,
+                rowGap: 3,
+                gridTemplateColumns: { xs: 'repeat(1, 1fr)', sm: 'repeat(1, 1fr)' }
+              }}
+            >
+              <RHFTextField name="firstname" label="First Name" />
+              <RHFTextField name="lastname" label="Last Name" />
+              <RHFDatePicker name="birthdate" label="Birth date" />
+              <RHFTextField name="email" label="Email Address" />
+              <RHFTextField name="phoneNumber" label="Phone Number" />
+            </Box>
+            <Stack direction="row" alignItems="flex-start" sx={{ mt: 3 }}>
+              <Checkbox checked={privacy} onChange={() => setPrivacy(!privacy)} />
```

Comment:
We have a `RHFCheckbox`, is there a reason you don't use this one?

Reply:
not required to submit